docs/api.spy.md

```diff
@@ -0,0 +1,27 @@
+# `uvu/spy`
+
+The `uvu/spy` module contains functions to spy on function calls and their arguments. You can use it to verify if a callback function is correctly invoked with the expected arguments.
+
+```js
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { spy } from 'uvu/spy';
+
+test('check if function is called twice', () => {
+  const testFunction = spy((a, b) => a + b);
```

Comment:
Shouldn't it be something more like this?

```suggestion

const add = (a, b) => a + b;

test('check if function is called twice', () => {
  const testFunction = spy(add);
```

Reply:
In what way would it change the test or the outcome? The way I see it, hosting the function out for this test doesn't affect our in any way and it's more of a stylistic choice.

Reply:
Yes but in that example in assertion you use 'add' function that doesn't exist. Might be misleading

Reply:
Ohh you're right, now that I'm back on a laptop I see the mistake. Sorry for being a bit grumpy earlier, that was a bit out of line from me.